plutus-contract/src/Plutus/Contract/Test/ContractModel.hs

```diff
@@ -503,13 +522,15 @@ setHandles a = State.modify (<> EmulatorAction (const a))
 
 instance ContractModel state => Show (StateModel.Action (ModelState state) a) where
     showsPrec p (ContractAction a) = showsPrec p a
+    showsPrec p (Unilateral w)     = showParen (p >= 11) $ showString "Unilateral " . showsPrec 11 w
 
 deriving instance ContractModel state => Eq (StateModel.Action (ModelState state) a)
 
 instance ContractModel state => StateModel (ModelState state) where
 
     data Action (ModelState state) a where
         ContractAction :: Action state -> StateModel.Action (ModelState state) ()
+        Unilateral :: Wallet -> StateModel.Action (ModelState state) ()
```

Comment:
These constructors could do with comments - what is `Unilateral`?

Reply:
nit: the grammar feels a touch off. It's more like "unilateralize"? Or maybe just "restrict participants"?